build/nlu-installer/src/installer.ts

```diff
@@ -61,30 +67,29 @@ export default async (argv: ArgV) => {
     throw new Error(`File ${argv.config} does not exist.`)
   }
 
-  let nlu: any
+  let nluVersion: string
   try {
     const fileContent = await fse.readFile(argv.config, 'utf8')
     const parsedContent = JSON.parse(fileContent)
-    nlu = parsedContent.nlu
+    nluVersion = parsedContent.nlu
```

Comment:
I learned recently that readJSON exists... So i'd suggest `nluVersion = (await fse.readJSON(argv.config)).nlu`